portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomFact() {
   const facts =
-      ['I am Jamaican', 'I am a virgo', ' I have been on Television before', 'I love the 90s movie Golden Girls']; 
+      ['I am a virgo '+ String.fromCodePoint(0x264d), ' I have been on Television before '+ String.fromCodePoint(0x1f4fa), 'I love the 90s show Golden Girls ' + String.fromCodePoint(0x1f475)]; 
```

Comment:
What's the "String.fromCodePoint(0x1f4fa)" thing?

Reply:
It converts emojis to string. I'll add a comment in the code.

Reply:
Ideally, you'd want these things to be global constants and a function, so we know what emoji it is.

const SAD_EMOJI = 0x222;
const CRYING_EMOJI = 0xffff;

function emojiString(code) {
  return String.fromCodePoint(blah);
}
//..
//..
//..
"I am a virgo" + emojiString(SAD_EMOJI) + " blah..."